v8-samples/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -stdlib=libc++")
```

Comment:
Hm, what is the intention of including this into this repo?

Reply:
I couldn't compile without it. Linking V8 libraries would lead to `undefined reference` errors.

Reply:
I meant the whole v8-samples directory, not just this line. ;)

Reply:
Oh right! Just needed it to test CMake scripts. Will remove it.